exercises/E-conditionals/exercise.js

```diff
@@ -6,8 +6,21 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
-var name = "Daniel";
-var danielsRole = "mentor";
+// var name = "Daniel";
+// var danielsRole = "mentor";
+
+function whoIsDaniel() {
+  var name = "Daniel";
```

Comment:
No Krish instead of Daniel? I thought you were cool man 😄😂

Reply:
Experimenting something new with my name bruv haha...Thanks for reviewing 👍🏻